addon/components/course-header.js

```diff
@@ -10,6 +10,7 @@ export default class CourseHeaderComponent extends Component {
 
   @restartableTask
   *changeTitle() {
+    this.courseTitle = this.courseTitle.trim();
```

Comment:
Now that the validator accounts for this is it needed here? If so maybe it should be on a setter for courseTitle instead?

Reply:
this is just to trim force input cleanup on submission, otherwise leading/trailing whitespace will remain in the form component until refresh (the backend will strip it out regardless).

feels weird/redundant to set it there, but couldn't think of a better place at that moment.